astropy/wcs/wcs.py

```diff
@@ -600,6 +600,7 @@ def calc_footprint(self, header=None, undistort=True, axes=None, center=True):
         Parameters
         ----------
         header : `~astropy.io.fits.Header` object, optional
+            Used to get ``NAXIS1`` and ``NAXIS2``
```

Comment:
That may be, but this seems overly specific.  I think all the user really cares about is that this is the FITS header of the image.